Resources/Prototypes/Entities/Objects/Weapons/Melee/baguette.yml

```diff
@@ -0,0 +1,17 @@
+- type: entity
+  name: baguette
+  parent: FoodBreadBaguette
+  id: WeaponBaguette
+  description: Bon appétit!
+  suffix: Weapon
+  components:
+  - type: MeleeWeapon
+    wideAnimationRotation: -135
```

Comment:
the rotation angle needs to be edited, now the animation does not quite match the angle of the sprite